rust/lance/src/index/vector/ivf/v2.rs

```diff
@@ -672,6 +661,57 @@ mod tests {
         );
     }
 
+    async fn test_remap(params: VectorIndexParams, nlist: usize) {
+        match params.metric_type {
+            DistanceType::Hamming => {
+                test_remap_impl::<UInt8Type>(params, nlist, 0..2).await;
+            }
+            _ => {
+                test_remap_impl::<Float32Type>(params, nlist, 0.0..1.0).await;
+            }
+        }
+    }
+
+    async fn test_remap_impl<T: ArrowPrimitiveType>(
```

Comment:
Does this only test the case where rows are deleted or does it also test the case where fragments are combined and row ids are changed?

Reply:
added